provider/provider_test.go

```diff
@@ -10,18 +10,18 @@ import (
 )
 
 var testProviders = map[string]terraform.ResourceProvider{}
+var testProvidersMocked = map[string]terraform.ResourceProvider{}
```

Comment:
I think the standard way of calling this is `testAccProviders` and `testMockProviders`